nbresult/__init__.py

```diff
@@ -49,4 +50,4 @@ def setUp(self):
         klass = self.__class__.__name__
         name = re.sub(r'(?<!^)(?=[A-Z])', '_', klass).lower()[len('test_'):]
         result_file = os.path.join(os.getcwd(), "tests", f"{name}.pickle")
-        self.result = pickle.load(open(result_file, 'rb'))
+        self.result = pd.read_pickle(result_file)
```

Comment:
The problem is that all `*.pickle` files are not `pandas.Dataframe` so what happens when it is something else?

Reply:
It should work with all the objects:
![image](https://user-images.githubusercontent.com/39556343/127296250-c24ec336-67d5-484e-9e1a-b996b2646157.png)

Example of a mean value (passed):
![image](https://user-images.githubusercontent.com/39556343/127296543-6b9df5d6-9164-4796-8fda-88b0665ea5e0.png)


Can we run a test on all the challenges with new version of the package before we communicate to students?